src/eduid/webapp/idp/schemas.py

```diff
@@ -89,3 +103,49 @@ class TouResponsePayload(EduidSchema, CSRFResponseMixin):
         version = fields.Str(required=False)
 
     payload = fields.Nested(TouResponsePayload)
+
+
+class UseOther1RequestSchema(IdPRequest):
+    username = fields.Str(required=False)  # optional username, if the user supplies an e-mail address
+    action = fields.Str(required=False)  # optional action, 'ABORT' or 'SUBMIT_CODE'
+    response_code = fields.Str(required=False)  # optional response code, if action == 'SUBMIT_CODE'
+
+
+class UseOther1ResponseSchema(FluxStandardAction):
+    class UseOther1ResponsePayload(EduidSchema, CSRFResponseMixin):
+        expires_in = fields.Int(required=True)  # to use expires_at, the client clock have to be in sync with backend
+        expires_max = fields.Int(required=True)  # to use expires_at, the client clock have to be in sync with backend
+        qr_img = fields.Str(required=True)  # qr_url as an inline img
+        qr_url = fields.Str(required=True)  # the link to where the user can manually enter short_code to proceed
+        short_code = fields.Str(required=True)  # six digit code for this request
```

Comment:
Can we rename short_code to something like other_device_auth_id?

Reply:
I don't see the other_device_ prefix as helping anything there, but sure we can rename it if we come up with a better name. I don't think auth_id is more indicative of the purpose of the code than short_code though :).

The purpose of the code is to give the human user the chance to verify that their two devices are... "working on the same state", to make sure they are not in the process of granting access to someone else than themselves. What's a good name for that?

Reply:
Maybe "shared_id"?

Reply:
shared_id I think

Reply:
agreed on display_id